src/ui/components/PasscodeModule/PasscodeModule.tsx

```diff
@@ -70,10 +71,9 @@ const PasscodeModule = ({
                           handleBiometricButton()
                           }
                         >
-                          <IonIcon
-                            slot="icon-only"
-                            className="passcode-module-fingerprint-icon"
-                            icon={fingerPrintSharp}
+                          <img
```

Comment:
We should show FaceId icon just if the biometry option is faceId, if not we should show fingerprint icon

Reply:
I updated.
### IOS
<img width="743" alt="Screenshot 2024-05-24 at 14 59 31" src="https://github.com/cardano-foundation/cf-identity-wallet/assets/162310763/e5e1483c-4bc4-412a-bb75-5bb922041757">

### Android
I cannot setup face auth for android I'm only capture icon with `fingerprint` mode
<img width="345" alt="Screenshot 2024-05-24 at 15 04 12" src="https://github.com/cardano-foundation/cf-identity-wallet/assets/162310763/e23e7e11-629c-435b-9bb0-707f1d6e90c8">